lib/vmdb/fast_gettext_helper.rb

```diff
@@ -2,6 +2,11 @@ module Vmdb
   module FastGettextHelper
     def self.human_locale(locale)
       # TRANSLATORS: Provide locale name in native language (e.g. English, Deutsch or Português)
+
+      # We lazy load the message catalogs on first use so we need to find another
+      # way to get a human friendly locale besides loading all of the catalogs
+      # to get this "human friendly locale" value.
+      return locale
```

Comment:
We need a way to provide human friendly locale names for the po files without reading, converting the po file to mo file, etc. for each po file, currently 4.  @himdel @Fryguy any suggestions here?